File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/articles_edited/provider/ArticlesEditedProvider.java
package org.wikiedufoundation.wikiedudashboard.course_detail.articles_edited.provider;

import org.wikiedufoundation.wikiedudashboard.helper.PresenterCallback;

public interface ArticlesEditedProvider {
    void requestArticlesEdited(String url, PresenterCallback presenterCallback);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/articles_edited/provider/ArticlesEditedProvider.kt
package org.wikiedufoundation.wikiedudashboard.course_detail.articles_edited.provider

import org.wikiedufoundation.wikiedudashboard.helper.PresenterCallback

interface ArticlesEditedProvider {
    fun requestArticlesEdited(url: String, presenterCallback: PresenterCallback<*>)
}
